Formula/m/mit-scheme.rb

```diff
@@ -93,7 +93,13 @@ def install
 
     ENV.prepend_path "PATH", buildpath/"staging/bin"
 
-    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}", "--without-x"
+    if OS.mac?
+      system "./configure", "--prefix=#{prefix}", "--mandir=#{man}", "--without-x", \
+        "CC=/usr/bin/gcc", "CXX=/usr/bin/g++"
+    else
+      system "./configure", "--prefix=#{prefix}", "--mandir=#{man}", "--without-x"
```

Comment:
With the ENV usage we can also do:

```suggestion
      system "./configure", "--mandir=#{man}", "--without-x", *std_configure_args
```